metricbeat/tests/system/test_http.py

```diff
@@ -33,10 +33,11 @@ def test_json(self):
         self.assertEqual(len(output), 1)
         evt = output[0]
 
-        assert evt["http"]["test"]["hello"] == "world"
+        assert evt["http"]["json"]["hello"] == "world"
```

Comment:
As the `namespace` is set to `test`, this should stay under test. Seems like your change does not respect this config anymore but should.

Reply:
Will this pass? Shouldn't this be under `test`?